agent/host_agent_test.go

```diff
@@ -544,7 +518,7 @@ var _ = Describe("Agent", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			runner = setupTestInfra(ctx, hostName, getKubeConfig().Name(), ns)
-			runner.CommandArgs["--bootstrap-kubeconfig"] = "/mgmt.conf"
+			runner.CommandArgs["--bootstrap-kubeconfig"] = "/root/.byoh/config"
```

Comment:
I thought we would remove `/root` hard coding?

Reply:
We have removed the hardcodings for kubeconfig container destination path[[here](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/pull/578/files#diff-86318bdbaa9a116307312ef026fe917b23a193098696192e43b30ea0b53c076cR195)] but this is the one test where we are explicitly passing the `--bootstrap-kubeconfig` flag. To remove the hardcoding here we have to do container Exec and fetch the User home dir.